tests/unit_tests/data_steward/utils/pipeline_logging_test.py

```diff
@@ -14,11 +14,38 @@
 import unittest
 import logging
 import mock
-import os
 from datetime import datetime
 
 # Project imports
-from utils.pipeline_logging import generate_paths, create_logger, setup_logger
+from utils.pipeline_logging import generate_paths, create_logger, setup_logger, get_logger
```

Comment:
at this point, i'd just import this as an alias and update the places these are called.

```suggestion
import utils.pipeline_logging as pl
```